lib/food_truck.rb

```diff
@@ -0,0 +1,34 @@
+class FoodTruck
+  attr_reader :name, :inventory
+
+  def initialize(name)
+    @name = name
+    @inventory = Hash.new(0)
+  end
+
+  def check_stock(item)
+    @inventory[item]
+  end
+
+  def stock(item, quantity)
+    @inventory[item] = quantity
```

Comment:
Small problem here... this will not account for adding more stock to the same item, this would override the stocked amount everytime you stock an object. I'm guessing your tests didn't try this... checking now.